spec/system/candidate_interface/course_selection/candidate_edits_course_choices_spec.rb

```diff
@@ -232,7 +232,7 @@ def then_i_should_see_the_review_page
   end
 
   def and_i_am_asked_to_select_site
-    expect(page).to have_content('Which location are you applying to?')
+    expect(page).to have_content('Which location are you interested in?')
```

Comment:
This system specs is the non continuous apps one, right?

Reply:
yes – but it uses the same locale